docs/guide/tables.md

```diff
@@ -0,0 +1,125 @@
+# Tables
+
+Since the SIX Web Components Library supports basic components only, we do not provide any table
+implementation. If your application depends on tables, you should consider using a 3rd party table
+implementation like [AG Grid](https://www.ag-grid.com/).
```

Comment:
I would be more specific and recommend 
- AG Grid for complex cases 
- plain vanilla HTML tables for simple tables (CSS to be provided from SIC5, too)
- and maybe even some hint about components that may be available for re-use SIX internally 

What do you think?